autonomi/src/client/data_types/scratchpad.rs

```diff
@@ -35,12 +35,21 @@ pub enum ScratchpadError {
 impl Client {
     /// Get Scratchpad from the Network
     /// It is stored at the owner's public key
-    pub async fn scratchpad_get(
+    pub async fn scratchpad_get_from_public_key(
```

Comment:
I don't like the name change here. Isn't it implied for the scratchpad that the address is derived from the public key? This function name would have made sense if there are multiple options. (E.g. `scrathpad_get_from_chunk_address`)